### DISCUSSION TITLE
Ollama options not displayed in chat

### DISCUSSION THREAD

**Discussion by Speedway1** [danny-avila/LibreChat#3238] [Troubleshooting] [answered]
### What happened?

I have configured Ollama in the librechat.yaml file but it's not appearing in the drop-down to choose from:

```
    - name: 'Ollama'
      apiKey: ''
      baseURL: 'https://192.168.0.173:3000/v1/chat/completions'
      models:
        default:
          [
            'samantha-1.1-westlake-7b:f16',
            'codestral:latest',
            'codesql:latest',
            'llama3:8b-instruct-fp16',
            'pearl:latest',
            'westlake:latest',
            'fusionnetmoe:latest',
          ]
        fetch: false
      titleConvo: true
      titleModel: 'samantha-1.1-westlake-7b:f16'
      summarize: false
      summaryModel: "current_model"
      forcePrompt: false
      modelDisplayLabel: 'Ollama'

```
did 
  npm run frontend
  npm run backend

See the screenshot, "Ollama" not there. Cannot find any of the Ollama models.

### Steps to Reproduce

1) Install librechat
2) edit .env
3) edit .yaml file and add Ollama section
4) run frontend then back end.
5) Go to URL in browser.
6) Attempt to select/find Ollama entries.

### What browsers are you seeing the problem on?

Chrome

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/100301611/9b9dfacf-5e6e-4c03-a15c-ee52e0e86cdc)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Add an api key for ollama: `apiKey: "ollama"`

**Reply by danny-avila** [reply, marked as answer]
example:

https://www.librechat.ai/docs/configuration/librechat_yaml/ai_endpoints/ollama

**Comment by Speedway1** [comment]
Thank you Danny, will do. Just wanted to quickly respond here to thank you for the rapid response from you.  We're pulling an all-nighter on  a project and this will help.

**Comment by Speedway1** [comment]
That has worked perfectly. One thing to add in case others are looking for it, is that https in the Ollama URL doesn't work. It needs to be http. But as it's locally hosted it should be secure behind all your firewalls or on your VPN so that shouldn't matter.

**Reply by berry-13** [reply]
well, technically you could use HTTPS to host Ollama. In the guide, the URL is shown as "**http**://localhost:11434/v1/chat/completions"
